ui/pages/v2/KustomizationDetail.tsx

```diff
@@ -0,0 +1,114 @@
+import _ from "lodash";
+import * as React from "react";
+import styled from "styled-components";
+import Flex from "../../components/Flex";
+import KubeStatusIndicator from "../../components/KubeStatusIndicator";
+import Link from "../../components/Link";
+import Page from "../../components/Page";
+import ReconciledObjectsTable from "../../components/ReconciledObjectsTable";
+import Text from "../../components/Text";
+import { useGetKustomization } from "../../hooks/automations";
+import { AutomationKind } from "../../lib/api/core/types.pb";
+import { formatURL } from "../../lib/nav";
+import { V2Routes, WeGONamespace } from "../../lib/types";
+
+type Props = {
+  name: string;
+  className?: string;
+};
+
+const Info = styled.div`
+  padding-bottom: 32px;
```

Comment:
It looks like the padding is the space for the Filter and Search things - will probs have to change later